### DISCUSSION TITLE
Project Zomboid server problem

### DISCUSSION THREAD

**Discussion by afus0** [GameServerManagers/LinuxGSM#3775] [Q&A] [answered]
Hello, i'm currently running CentOS 7 on a VPS. It's been 2 days that i'm actually trying to figure out how to fix this problem with the game "Project Zomboid", i used to have no problems with other game servers.
I cant make work this one.
Here's what i get from the log while trying to run it:

> [05-02-22 14:12:13.388] LOG  : General     , 1644066733388> 8,227,065,840> cachedir set to "/home/pzserver/Zomboid".
> [05-02-22 14:12:13.421] ERROR: General     , 1644066733421> 8,227,065,873> java.lang.UnsatisfiedLinkError: /home/pzserver/serverfiles/natives/libPZClipper64.so: /lib64/libstdc++.so.6: version `CXXABI_1.3.8' not found (required by /home/pzserver/serverfiles/natives/libPZClipper64.so).
> [05-02-22 14:12:13.422] ERROR: General     , 1644066733422> 8,227,065,875> 	at java.base/jdk.internal.loader.NativeLibraries.load(Native Method).
> [05-02-22 14:12:13.423] ERROR: General     , 1644066733423> 8,227,065,875> 	at java.base/jdk.internal.loader.NativeLibraries$NativeLibraryImpl.open(Unknown Source).
> [05-02-22 14:12:13.425] ERROR: General     , 1644066733425> 8,227,065,877> 	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(Unknown Source).
> [05-02-22 14:12:13.426] ERROR: General     , 1644066733425> 8,227,065,878> 	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(Unknown Source).
> [05-02-22 14:12:13.426] ERROR: General     , 1644066733426> 8,227,065,878> 	at java.base/jdk.internal.loader.NativeLibraries.findFromPaths(Unknown Source).
> [05-02-22 14:12:13.427] ERROR: General     , 1644066733427> 8,227,065,879> 	at java.base/jdk.internal.loader.NativeLibraries.loadLibrary(Unknown Source).
> [05-02-22 14:12:13.427] ERROR: General     , 1644066733427> 8,227,065,879> 	at java.base/java.lang.ClassLoader.loadLibrary(Unknown Source).
> [05-02-22 14:12:13.428] ERROR: General     , 1644066733428> 8,227,065,880> 	at java.base/java.lang.Runtime.loadLibrary0(Unknown Source).
> [05-02-22 14:12:13.428] ERROR: General     , 1644066733428> 8,227,065,880> 	at java.base/java.lang.System.loadLibrary(Unknown Source).
> [05-02-22 14:12:13.429] ERROR: General     , 1644066733429> 8,227,065,881> 	at zombie.vehicles.Clipper.init(Clipper.java:24).
> [05-02-22 14:12:13.429] ERROR: General     , 1644066733429> 8,227,065,881> 	at zombie.network.GameServer.main(GameServer.java:326).
> 

Is there anyone that know how to fix this? thanks in advance

**Comment by h3o66** [comment, marked as answer]
I do not know any easy solution.
You probably need to build a newer `libstdc++.so.6` and put it into a new path variable for this server.
The PZ developers do not seem to support CentOS 7
